### PR TITLE
HOMS-326 Update README instruction for developers

### PR DESCRIPTION
**Pull request by NatalieKor**

### CODE REVIEW

File: WITHOUT_ORACLE.md [hunk 1]
```diff
@@ -46,14 +54,12 @@ If you don't want to use Oracle as source for your HOMS instance.
...
     docker-compose -f docker-compose.dev.yml up -d
     ```
 
-9. Navigate to [Camunda Admin](http://localhost:8080/camunda) and create admin user with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file.
-
-10. (Optional) If you want to use demo processes, navigate to [Camunda](http://localhost:8080/camunda/app/admin/default/#/user-create) and create user with `user@example.com` email.
+10. Navigate to [Camunda Admin](http://localhost:8766/camunda) and create admin user with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file.
 
-11. Log in at [HydraOMS](http://localhost:3000) with *`user@example.com`*/*`changeme`*.
+11. (Optional) If you want to use demo processes, navigate to [Camunda](http://localhost:8766/camunda/app/admin/default/#/user-create) and create user with `user@example.com` email.
```
Comment (dolfinus) [inline WITHOUT_ORACLE.md]: You can remove this line. Seed application in camunda docker image creates user with certain email automatically https://github.com/latera/camunda-ext/blob/master/seed/src/main/groovy/org/camunda/latera/bpm/configuration/SeedApplication.groovy#L38

Comment (dolfinus) [inline WITHOUT_ORACLE.md]: You can remove these lines. Seed application in camunda docker image creates user with proper credentials automatically https://github.com/latera/camunda-ext/blob/master/seed/src/main/groovy/org/camunda/latera/bpm/configuration/SeedApplication.groovy#L38

Comment (TimAle) [reply WITHOUT_ORACLE.md]: I think we should remove this as we did for general manual.

File: WITHOUT_ORACLE.md [hunk 2]
```diff
@@ -46,14 +54,12 @@ If you don't want to use Oracle as source for your HOMS instance.
...
+9. Run `docker-compose` with custom config:
 
     ```bash
     docker-compose -f docker-compose.dev.yml up -d
     ```
 
-9. Navigate to [Camunda Admin](http://localhost:8080/camunda) and create admin user with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file.
-
-10. (Optional) If you want to use demo processes, navigate to [Camunda](http://localhost:8080/camunda/app/admin/default/#/user-create) and create user with `user@example.com` email.
+10. Navigate to [Camunda Admin](http://localhost:8766/camunda) and create admin user with credentials equal to `BPM_USER:BPM_PASSWORD` values from `.env` file.
```
Comment (TimAle) [inline WITHOUT_ORACLE.md]: Also we can remove this and add something like 
```
You can login to Camunda Admin inteface with credentials equal to BPM_USER:BPM_PASSWORD values from .env file (user/changeme if these variables aren't set).
```
I mean, let's sync it with general manual.

File: WITH_ORACLE.md [hunk 3]
```diff
@@ -81,14 +86,12 @@ If you want to use Oracle as source for your HOMS instance.
       password: <%= ENV['HOMS_DB_PASSWORD'] %>
     ```
 
-12. Run `docker-compose` with custom config:
+13. Run `docker-compose` with custom config:
 
     ```bash
     docker-compose -f docker-compose.dev.oracle.yml up -d
```
Comment (TimAle) [inline WITH_ORACLE.md]: I think we should use `docker-compose.dev.yml` here since we run HOMS locally. So `docker-compose.dev.yml` will be redundant.

**Pull request merged**